content/sensu-go/5.20/sensuctl/reference.md

```diff
@@ -468,15 +468,34 @@ To export only handlers and filters to a file named `my-handlers-and-filters.yam
 sensuctl dump handler,filter --format yaml --file my-handlers-and-filters.yaml
 {{< /highlight >}}
 
-### sensuctl dump resource types
+### sensuctl describe-type resource types
 
-You can use the sensuctl `--types` subcommand to list the types of supported resources:
+{{% notice important %}}
+**IMPORTANT**: The `sensuctl describe-type` command deprecates `sensuctl dump --types`.
+{{% /notice %}}
+
+Use `sensuctl describe-type` to list the types of supported resources.
+For example, to list all types:
+
+{{< highlight shell >}}
+sensuctl describe-type --all
```

Comment:
`all` is actually an argument and not a flag, so it's going to be `sensuctl describe-type all`.